db/sql/150_stop_correspondence_schema.sql

```diff
@@ -50,6 +52,41 @@ COMMENT ON COLUMN observation.lat IS
 COMMENT ON COLUMN observation.geom IS
 'Observation POINT geometry in ETRS-TM35 coordinate system, generated from long & lat.';
 
+CREATE OR REPLACE FUNCTION stopcorr.refresh_observation() RETURNS bigint AS $func$
+DECLARE
+  n_inserted bigint;
+BEGIN
+  DELETE FROM observation;
+  WITH ins AS (
+    INSERT INTO observation (tst, event, oper, veh, route, dir, oday, start, stop_id, long, lat)
+    SELECT
+      hfp.tst,
+      hfp.event_type,
+      hfp.observed_operator_id,
+      hfp.vehicle_number,
+      hfp.route_id,
+      hfp.direction_id,
+      hfp.oday,
+      hfp.start,
+      hfp.stop,
+      hfp.longitude,
+      hfp.latitude
+    FROM hfp.view_as_original_hfp_event hfp
+    WHERE (
+      hfp.event_type = 'DOO' OR
+      hfp.event_type = 'DOC'
+    ) AND
+    hfp.longitude IS NOT NULL AND
+    hfp.latitude IS NOT NULL
+    ON CONFLICT DO NOTHING
```

Comment:
Got some primary key conflicts so I added this. I suppose it's because some events happened during the same second.